samples/node/RedisOutputBinding/extensions.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
```

Comment:
The csproj allows us to use custom extensions. For customers, they won't need to use this because they'll be using the extensions from the extensions bundle. For us, since the extension is built locally, we have to reference the extension that we're using in the sample itself so that the samples can use the locally built extension